integration/resources/templates/combination/api_with_custom_domains_regional_latency_routing.yaml

```diff
@@ -0,0 +1,56 @@
+Parameters:
```

Comment:
Please add both templates as transform tests as well, so that it's clear to what it transforms:

```
python bin/add_transform_test.py --template-file integration/resources/templates/combination/api_with_custom_domains_regional_latency_routing.yaml
python bin/add_transform_test.py --template-file integration/resources/templates/combination/api_with_custom_domains_regional_latency_routing_ipv6.yaml
```